.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,5 +27,6 @@ I have:
 - [ ] Added or updated unit **and** E2E tests for my change.
 - [ ] Run `make reviewable` to ensure this PR is ready for review.
 - [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.
+- [ ] Opened a PR updating the [docs](https://docs.crossplane.io/contribute/contribute/), if necessary.
 
-[contribution process]: https://git.io/fj2m9
\ No newline at end of file
+[contribution process]: https://git.io/fj2m9
```

Comment:
this looks like a no-op change?

Reply:
Just the newline, my editor configuration removes them from the end of some types of files.